ts/components/screens/BaseScreenComponent/index.tsx

```diff
@@ -215,7 +206,16 @@ const BaseScreenComponentFC = React.forwardRef<ReactNode, Props>(
           // TODO: remove local feature flag
           if (zendeskEnabled) {
             return () => {
-              dispatch(zendeskSupportStart());
+              dispatch(
+                zendeskSupportStart({
+                  // If contextualHelpConfig is undefined this function is not called
```

Comment:
This is a weak assumption. From this point we are propagating some data that we know are defined until we'll do a strong assertion 
![Schermata 2021-12-13 alle 19 30 25](https://user-images.githubusercontent.com/822471/145868182-335e9f54-fe27-4722-9308-3b0939744703.png)

My suggestion is to consider and use them as defined by increasing your types
Here an attempt that tries to make this logic stronger
4cab2c8740911b6a187e42eae7566ea543f7abb5

Do you think it could be an improvement?

Reply:
I'm not sure your solution work for every case we have.
Take into account that we can start the Zendesk workflow also outside the `BaseScreeComponent`.
How do you think about this solution: [3b5f262](https://github.com/pagopa/io-app/pull/3572/commits/3b5f26204ec2af24c2d7cd7132a71107e1303f72)? (with this fix: [90841e6](https://github.com/pagopa/io-app/pull/3572/commits/90841e66e2f3399dbb135f31d6712678e65266dc))

Reply:
I guess it is even better!